### ISSUE TITLE
org-toggle-pretty-entities does not work in visualize buffer

### ISSUE THREAD

**Issue by santoshs** [Kungsgeten/org-brain#273]
I can toggle inline images using org-toggle-inline-images, but org-toggle-pretty-entities does not work. i am not sure how to debug it further, but if anything I can do, I could be of help.

**Comment by Kungsgeten** [comment]
For me pretty entities work, but I have them enabled by default in `org-mode`. However if I run `org-toggle-pretty-entities` they turn off in `org-brain-visualize`, and then I can not turn them back on.

I think this is because `org-brain` opens the entry text in a temporary buffer, and runs `org-mode` there to get the text. Since I have pretty entities on by default, `font-lock` will transform the characters. If doing the same in `org-brain` there are no code searching for `org-pretty-entities`, so they won't be displayed when updating `font-lock`.

Long story short is that `org-brain` probably will not support toggling pretty entities. I've made a commit though that will show pretty entities in entry text. I think that's a sensible default. Now `org-toggle-pretty-entities` will turn it off, but it can't be turned back on that way (you could revert the buffer by pressing `g` though).

**Comment by santoshs** [comment]
Yep, pretty entities as a default in the entry text is good enough. Thanks for your super speedy response :-)
